### DISCUSSION TITLE
KubeCon + CloudNativeCon Europe | Takeaways

### DISCUSSION THREAD

**Discussion by muthuannamalai12** [EddieHubCommunity/support#1466] [Conference Notes]
As a student, I was puzzled to find which sessions I need to attend at  KubeCon + CloudNativeCon Europe. Then I saw the "College to Cloud Native" session which cleared all my doubts.

Sessions a Student can attend at KubeCon: 
1. From Allyship to Sponsorship: Solutions to Design a Diverse and Inclusive Community
2. Peer Group Mentoring + Career Networking 
3. Business Value and 101 tracks 
4. Path To Non-code contribution and A Pathway to CNCF Citizenship

You can watch the video here: https://www.youtube.com/watch?v=E_iiOet6J8Q

**Comment by kaiwalyakoparkar** [comment]
KubCon is for everyone and an absolute beginner can also become part of it and increase it's network

**Reply by muthuannamalai12** [reply]
Absolutely True

**Reply by kaiwalyakoparkar** [reply]
I will keep on updating this conversation as I attend more sessions

**Reply by muthuannamalai12** [reply]
I will too update it with my takeaways

**Reply by kaiwalyakoparkar** [reply]
Sure I we would love to read each other's take aways

**Comment by eddiejaoude** [comment]
awesome! thanks for sharing 👍

**Comment by arizon-dread** [comment]
I scheduled a bunch of project office hours amd was disappointed. Except for helm, that session was great!

**Reply by arizon-dread** [reply]
The openshift commons space was also really good

**Comment by kaiwalyakoparkar** [comment]
The session on "Inside networking of Kubernetes" was really very informative and descriptive. It demonstrated how the network is established in the different parts of the node, Kubernetes, pod, etc.

**Comment by muthuannamalai12** [comment]
The session on We Didn’t Start the Fire: Communication Breakdowns and How to Prevent Them is really inspirational. The speakers shared their failures and mistakes they did. Today I learned the importance of communication in this world and also learned about how a communication breakdown would like.
